lib/sentry.server.js

```diff
@@ -0,0 +1,20 @@
+import * as Sentry from '@sentry/node'
+import { Dedupe, ExtraErrorData, RewriteFrames, Transaction } from '@sentry/integrations'
+
+export default function (ctx, inject) {
+  const opts = Object.assign({}, <%= serialize(options) %>, {
+    integrations: [
+        new Dedupe(),
+        new ExtraErrorData(),
+        new RewriteFrames(),
+        new Transaction()
+      ]
+  })
+
+  if (!opts.disabled) {
+    Sentry.init(opts)
```

Comment:
Is this init happening once or per-request? If sentry does not allows instance creation maybe it would be better to initialize it outside of this function and just inject it. (or move init to module and just inject here)

Reply:
Exported plugin function is called per ssr request so this doesn't seem correct. And as you mentioned in other comment, Init is duplicated here and in the module for server.